packages/opentelemetry/src/utils/contextData.ts

```diff
@@ -0,0 +1,36 @@
+import type { Context } from '@opentelemetry/api';
+import type { Hub, PropagationContext } from '@sentry/types';
+
+import { SENTRY_HUB_CONTEXT_KEY, SENTRY_PROPAGATION_CONTEXT_CONTEXT_KEY } from '../constants';
+
+/**
+ * Try to get the Propagation Context from the given OTEL context.
+ * This requires the SentryPropagator to be registered.
+ */
+export function getPropagationContextFromContext(context: Context): PropagationContext | undefined {
```

Comment:
I expose these as functions instead of exposing the constants, which should encapsulate this better.